.env.example

```diff
@@ -0,0 +1,5 @@
+# Assumes the Postgres server was started using the Compose file in docker/postgres/docker-compose.yml
+DATABASE_URL=postgres://postgres:@localhost:5432/compass
+
+GOOGLE_CLIENT_ID=foo
```

Comment:
Are these needed to the get the app working?

Reply:
not currently, good point

I'll leave a note